### ISSUE TITLE
Endpoint Error. Some node are return error in transactionStatus endpoint.

### ISSUE THREAD

**Issue by AnthonyLaw** [nemtech/catapult-rest#496]
Error Message
```
"code": "Internal",
"message": "error retrieving data for id: '7C8AC7552BC0F3909A3F4D236F2AC7921A1063A8A22D56674BC09FA088ACEBC4' (length 2)"
```

Transaction Hash: 7C8AC7552BC0F3909A3F4D236F2AC7921A1063A8A22D56674BC09FA088ACEBC4

Transaction detail: http://api-01.eu-central-1.0.10.0.x.symboldev.network:3000/transactions/confirmed/7C8AC7552BC0F3909A3F4D236F2AC7921A1063A8A22D56674BC09FA088ACEBC4

Node return **Error**: http://api-01.eu-central-1.0.10.0.x.symboldev.network:3000/transactionStatus/7C8AC7552BC0F3909A3F4D236F2AC7921A1063A8A22D56674BC09FA088ACEBC4

Node return **Success**: http://api-01.us-west-1.0.10.0.x.symboldev.network:3000/transactionStatus/7C8AC7552BC0F3909A3F4D236F2AC7921A1063A8A22D56674BC09FA088ACEBC4

Node is synced, it should return same result.

**Comment by Vektrat** [comment]
not a rest error, the database contains a duplicate status value

```
[{
	"hash": "7C8AC7552BC0F3909A3F4D236F2AC7921A1063A8A22D56674BC09FA088ACEBC4",
	"code": "Failure_LockHash_Unknown_Hash",
	"deadline": "28247489983",
	"group": "failed"
}, {
	"group": "confirmed",
	"code": "Success",
	"hash": "7C8AC7552BC0F3909A3F4D236F2AC7921A1063A8A22D56674BC09FA088ACEBC4",
	"deadline": "28247489983",
	"height": "20500"
}]
```

@Jaguar0625

**Comment by Vektrat** [comment]
reopening because apparently as mentioned in slack https://nemgroup-workspace.slack.com/archives/C015AEQ9ZL2/p1601672504284100 it is possible for a hash to be listed in different statuses at once, so we might need to return an array of statuses instead of a single object

**Comment by Vektrat** [comment]
clients are interested in returning just a single value, and gimre agreed to this approach, by following these conditions:

```
confirmed > unconfirmed > custom_statuses(includes partial) > failed
```

so we will proceed with this implementation to return just one status even though the hash may appear in different collections

**Comment by rg911** [comment]
closed via #564
